GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,97 @@
+# Cubyz is a Sandbox Game, not a Survival Game
+The goal of this project is to create a game about exploration, adventure, and building. There is no looming threat of hunger or monsters; the player should be able to build in peace and choose how they would like to play. 
+
+### Avoid Unavoidable Threats
+A player should have to create or approach a threat on their own terms, rather than the threat approach them. This should allow the player to interact with whatever they want without feeling forced to.
+
+**Examples:**
+- A player will not be attacked by monsters at night. They have to actively approach a monster-infested place such as a structure or cave. This is so they have free reign to do activites during the night without being interrupted or frustrated.
+- A player has lost much health, and must risk what little life they have left to gather food. Note that the player cannot heal unless they have an accessory that consumes their energy. This is a situation the player has to face because of their own skill level.
+- A player summons an event that spawns waves of enemies to their base.
+
+### Progression = Risk
+In order to progress in the game, the player must take risks. The player is otherwise free to stay where they are in terms of progression.
+
+**Examples:**
+- The deeper the player goes into the world's caves, the more valuables they can find, but consequentially, the more monsters they'll come across.
+- The player must defeat an enemy, boss, or event to obtain an item that helps them progress.
+- In a multiplayer world, a player might have to attack an enemy player to rise up in power.
+
+# Break the Cycle
+### No Dimensions
+Instead of creating seperate dimensions, we can fit these places physically into Cubyz' massive world to allow the player to come across them on their own.
+"Dimensions" such as an underworld and an overworld are already planned for the game.
```

Comment:
I would like to explictly see a mention of the root and sky islands here instead of the vague terms "underworld"/"overworld".